utils/docker-entrypoint.sh

```diff
@@ -2,4 +2,5 @@
 
 set -e
 
+sudo -E /tini -g -s -- /usr/local/bin/apache-expose-envvars.sh $@;
```

Comment:
this can certainly by put in the `docker-entrypoint-as-root.sh` file instead (and without the "sudo -E /tiny" part.

Also, this needs to be wrapped in a "if" statement to be triggered only for Apache variants.